shells/tests/artifacts/Common/Detail.manifest

```diff
@@ -0,0 +1,18 @@
+// @license
```

Comment:
should it be `.arcs` file?

PS: i just realized that we still have a bunch of these in our codebase. probably ok to rename in a separate PR, whatever you prefer.

Reply:
I think that makes sense. 

The PR was getting huge so I tried to limit the cleanup work (not totally successful).

Reply:
+1 to renaming in another PR.